dirs/dirs.go

```diff
@@ -260,6 +260,11 @@ func SetRootDir(rootdir string) {
 
 	SystemApparmorDir = filepath.Join(rootdir, "/etc/apparmor.d")
 	SystemApparmorCacheDir = filepath.Join(rootdir, "/etc/apparmor.d/cache")
+	if _, err := os.Stat(SystemApparmorCacheDir); os.IsNotExist(err) {
```

Comment:
this could be !osutil.FileExists (it really means both exist and access but given who runs this it should be fine)

Reply:
Done